### DISCUSSION TITLE
time sync

### DISCUSSION THREAD

**Discussion by iccor56** [abraunegg/onedrive#2169] [Q&A] [answered]
i use free file sync with one drive.

is there a way to adjust the time stamps when received or sent so that read as the same time on both ubuntu and windows?
(the whole core clock in local vs gmt issue)

**Comment by abraunegg** [comment, marked as answer]
> is there a way to adjust the time stamps when received or sent so that read as the same time on both ubuntu and windows?

You need to make sure that your Windows system + your Ubuntu system have the correct timezone configured.

The timestamps are set correctly based on the Operating System and File System time. The UTC|GMT time is used for file compare.

If you are seeing different timestamps - this will be because your configured timezone in your operating system is not what you think it is.
